src/Form/FormAutosuggest.jsx

```diff
@@ -102,6 +102,7 @@ function FormAutosuggest({
   const iconToggle = (
     <IconButton
       className="pgn__form-autosuggest__icon-button"
+      data-testid="autosuggest_iconbutton"
```

Comment:
We recently [rewrote all tests](https://github.com/openedx/paragon/pull/2581) (except `FormAutosuggest`) to RTL. For uniformity of test identifiers, we agreed to use a kebab case. Can you convert the names of the identifiers to a uniform format? `autosuggest-iconbutton`